packages/babel-plugin-transform-classes/test/fixtures/extend-builtins/loose/options.json

```diff
@@ -1,6 +0,0 @@
-{
-  "plugins": [
-    ["transform-classes", { "loose": true }],
-    "transform-block-scoping"
-  ]
-}
```

Comment:
This test is renamed to `assumption-superIsCallableConstructor/extend-builtins`.